packages/app/src/cli/services/dev/update-extension.ts

```diff
@@ -68,6 +65,24 @@ export async function updateExtensionDraft({
   }
 }
 
+export async function updateMultipleExtensionDrafts({
+  extension,
+  developerPlatformClient,
+  apiKey,
+  registrationId,
+  stdout,
+  stderr,
+}: UpdateExtensionDraftOptions) {
+  const registrationIdArr = registrationId.split(',')
```

Comment:
this is such a hack lol.. basically if there are multiple registration ids, the data shape is something like:
```
{ webhook-subscription: '1250,1251'}
```
(bc I just turned the entire array into a string in `mapExtensionsIdsNonUuidManaged`)
So the `split` here turns this back into an array

`[1250, 1251]`